### DISCUSSION TITLE
Testing failed: Expansion of macro 'PerceptionIgnored()' did not produce a observing accessor

### DISCUSSION THREAD

**Discussion by rehatkathuria** [pointfreeco/swift-perception#44] [General]
```
 @PerceptionIgnored private var key: Window? { windows.first(where: { $0.isKeyWindow }) }
```

A test that uses an object which has the following declared is causing the test to fail

**Comment by rehatkathuria** [comment]
Where `Window` is a typealias for `UIWindow` and `NSWindow` respective of the platform

**Reply by mbrandonw** [reply]
Can you please provide some code that actually demonstrates the problem? It's difficult to provide help without that. Also computed properties don't need `@PerceptionIgnored` as they are not observable. Only stored properties are observable.
